code/obj/item/mono_card_game.dm

```diff
@@ -0,0 +1,83 @@
+/obj/item/card_group/mono
+	card_style = "mono"
+	total_cards = 108
+	card_name = "MONO"
+	var/numToSpecial = list("10"="Block", "11"="Reverse", "12"="Draw 2")
+
+	New()
+		..()
+		//Add special cards
+		for(var/i = 1, i<5, i++){
+			var/obj/item/playing_card/wild = new /obj/item/playing_card(src)
+			var/obj/item/playing_card/fuckYou = new /obj/item/playing_card(src)
+			stored_cards += wild;
+			stored_cards += fuckYou;
+			setMonoCard(wild, 5, 0);
+			setMonoCard(fuckYou, 5, 1);
+		}
+		//Add one set of zeroes
+		for(var/i = 1, i < 5, i++){
+			var/obj/item/playing_card/card = new /obj/item/playing_card(src)
+			stored_cards += card
+			setMonoCard(card, i, 0)
+		}
+		//Do rest of the numbers
+		for(var/i=1, i<5, i++){
+			for(var/j = 1, j<13, j++){
+
+				var/obj/item/playing_card/card = new /obj/item/playing_card(src)
+				stored_cards += card
+				setMonoCard(card, i, j)
+
+				var/obj/item/playing_card/card2 = new /obj/item/playing_card(src)
+				stored_cards += card2
+				setMonoCard(card2, i, j)
+			}
+		}
+		update_group_sprite()
+
+	proc/setMonoCard(var/obj/item/playing_card/target , var/suit, var/num)
```

Comment:
Why not make Mono cards a subtype of playing_card and make this the constructor?

Reply:
Feedback pushed!